### DISCUSSION TITLE
BluetoothLE HCI RAW H4 Question

### DISCUSSION THREAD

**Discussion by HanfG** [zephyrproject-rtos/zephyr#69922] [Q&A]
I have a project that has BLE features. I's using ***`nRF52832` running Zephyr HCI RAW H4*** as HCI controller in the system. HCI Host was runing btstack.
Here is my question:
Btstack can not connect to multiple peripherals using gap_whitelist_* functions.
Zephyr send event "ACL Connection Already Exists" when  I call gap_whiltelist_add in the second time.
I talked to the author of btstack about this problem and he analyzed the hci raw logs and thought it was a zephyr issue. 
Here is (the link)[https://groups.google.com/g/btstack-dev/c/sRN4yoAmDZk] to the discussion.
I'm sure I'm using the latest release of zephyr. Zephyr app was same as [`samples/bluetooth/hci_uart` ](https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/bluetooth/hci_uart).

**Comment by mringwal** [comment]
Hi Hanf. Could you also upload the HCI dump file? You can attach it here if you first zip it.
